scripts/run.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 echo "setting up kubeconfig"
 ./usr/local/app/scripts/init_kubeconfig.sh
+
+if NGINX_ENABLED; then
```

Comment:
nit: this env var may be more descriptive if we call it `RELEASE_MIRROR_ENABLED`